Migration-Project-Guides/GoogleToGoogle.md

```diff
@@ -210,3 +275,28 @@ CloudM Migrate has several feature to enhance the migration, these can be config
 
 There are <a href="https://cloudm-migrate.github.io/documentation/Engineering-Reference/ProjectAdvancedOptions.html#email-attachment-to-drive">several features</a> for manipulating attachments while migrations are inflight. This includes removing attachments from the emails and replacing them with MyDrive links to the former attachments. Migrated documents can also optionally be shared with the recipients of the original email. This can dramatically shrink mailboxes on the destination.
 
+## Unsupported Data Types for Migration
+[Back to Top](#top)
+
+- Mail: Spam label
+- Mail: Categories
+- Mail: Filters
+- Mail: Settings
+- Groups: Messages
+- Groups: Settings
+- Calendar: Subscriptions
+- Calendar: Settings
+- Calendar: Appointment Slots
+- Contatcs: Directory
+- Contacts: Fields
+- Drive: File/folder creation date (will be reset to the date the file was migrated)
+- Drive: Shared with data (will be reset to the date the file was migrated)
+- Drive: Trash/Deleted Items
+- Drive: Revision history of Google format files
+- Drive: Externally owned files
+- Drive: Google My Maps
+- Drive: (New) Google Sites
+- Drive: Google Data Studio
+- Drive: Any third party shortcuts added using "Connect more apps"
+
```

Comment:
Do we want to talk about what can be migrated with vault as this list changes like deleted items.

Reply:
Vault is a separate endpoint so will have its own guide